.eslintrc

```diff
@@ -32,6 +32,7 @@
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
Can you recall what prompted this?

Reply:
Yes, I was accessing `HTMLElement.shadowRoot` using `someElement.shadowRoot!`. The `shadowRoot` property is nullable in general, but in the specific context it should never be.